targets/python/Archive/PlayFabUtil.py.ejs

```diff
@@ -0,0 +1,57 @@
+import json
+<% if (hasClientOptions) {
+%>import PlayFabClientModels
+<% } else if (hasServerOptions) {
+%>import PlayFabServerModels
+<% } %>
+
+class PlayFab:
```

Comment:
We are targeting py2.7 right? Any reason to not use new style classes (ie extend object)?

Reply:
We were attempting to extend base objects. This particular Util file wasn't actually ever used for Python (It was needed in other sdk's, hence why I ported it).